apps/ai-hero/src/app/page.tsx

```
@@ -86,20 +90,94 @@
+			validCoupon = coupon
+		}
+	}
+
+	const allProducts = await db.select().from(products)
+	const productIds = allProducts.map((p) => p.id)
+
+	let defaultCoupon: any = null
+	if (productIds.length > 0) {
+		const defaultCoupons =
+			await courseBuilderAdapter.getDefaultCoupon(productIds)
+		if (defaultCoupons && defaultCoupons.defaultCoupon) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)